backend/src/models/SavedApplication.ts

```diff
@@ -1,35 +1,41 @@
 import { InferSchemaType, Schema, model } from "mongoose";
 
-const savedApplicationSchema = new Schema({
-  userId: {
-    type: String,
-    required: true,
+const savedApplicationSchema = new Schema(
+  {
+    userId: {
+      type: String,
+      required: true,
```

Comment:
Shouldn't be too big of a deal since this is just a small change but for best practices can you run `git pull --rebase origin main`? There are some redundant changes here because your local commit tracking doesnt know that the previous commits you've made has already been pushed into `origin/main`, so in this case rebase would update your local commit history to be on track with `origin/main`. For more information you can read into [here](https://git-scm.com/docs/git-rebase).